core/src/main/java/org/infinispan/commands/CommandInvocationId.java

```diff
@@ -60,10 +61,7 @@ public int hashCode() {
 
    @Override
    public String toString() {
-      return "CommandUUID{" +
-            "address=" + address +
-            ", id=" + id +
-            '}';
+      return "InvocationId:" + Objects.toString(address, "local") + ":" + id;
```

Comment:
I've tried to shorten it because I want to put it in every log line, via the MDC. Still, I guess I can use the full class name until we come up with something catchier.